py/desitarget/mock/sfdmap.py

```diff
@@ -0,0 +1,342 @@
+# Licensed under an MIT "Expat" license - See LICENSE
```

Comment:
Looks like you got this from https://github.com/kbarbary/sfdmap (good) but didn't bring over the license or credit (bad).  Please add that as a comment in this file, including the URL and the date and githash of that repo for when you copied it (probably bacdbbd).